functions/src/utils.js

```diff
@@ -49,3 +49,10 @@ exports.typesenseDocumentFromSnapshot = async (
 
   return typesenseDocument;
 };
+
+exports.importDocumentsConfig = (batchSize) => ({
+  action: "emplace",
+  dirty_values: "coerce_or_drop",
+  return_doc: false,
+  batch_size: batchSize.length,
```

Comment:
The batch_size here controls [server-side batching](https://typesense.org/docs/guide/syncing-data-into-typesense.html#handling-rejecting-write-running-out-of-resource-type-errors:~:text=%23-,Client%2Dside%20batch%20size%20vs%20server%2Dside%20batching,-In%20the%20import), and we do not want to change this from the default in most cases. 

Could you remove this?